.chloggen/jaegerremotesampling_replace_thrift-gen_with_proto-gen.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: jaegerremotesampling
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add featuregate to replace Thrift-gen with Proto-gen types for sampling strategies
```

Comment:
Very dumb question: the extension is still usable w/ the deprecation? And users can pick between thrift or proto in their extension config (or otelcollector config)?

Reply:
(documentation updates would probably clarify)